### ISSUE TITLE
Class App\Http\Controllers\Laralum\Laralum\Controllers\LaralumController does not exist

### ISSUE THREAD

**Issue by nexxai** [Laralum/Laralum#34]
```
➜  project git:(master) ✗ php artisan route:list

In Container.php line 752:
                                                                                           
  Class App\Http\Controllers\Laralum\Laralum\Controllers\LaralumController does not exist  
```
                                                                                           
Hey guys, so I don't exactly know what I did since I haven't touched anything related to Laralum in months, but I went to print a route list and I'm getting the above error. The research I've done suggests that it might be a namespace issue (which would explain why it thinks it's supposed to look in App\Http\Controllers instead of the Vendors tree) but I can't figure out what I possibly could have changed that would do this.

One thing to note is I don't run the route:list artisan command very often so this may have been from a long time ago that I'm just now noticing, but any direction to where to start looking would be great.

I've already grep -r'd the entire code tree for anything obvious but I'm not finding anything that would suggest a problem.

**Comment by 24aitor** [comment]
Mmmh Grep is only a file searcher, should not modify anithing.

Could you check the real name of Laralum project folder inside the vendor's folder?

Nevertheless a fresh laralum installation, doesn't cause any error (ATM)

**Comment by nexxai** [comment]
Sorry it took me so long to get back to you but I've been super busy with work just before the Christmas break.  

I tried removing and reinstalling the package, and it made no difference; still the exact same error message.

What I *did* discover however was that I had added the routes in my web.php file at some point (a while ago), but it must have been "fighting" with the ones that are included with the package.  I commented them out from the web.php file and everything seems to be working properly now.
